sklearn/metrics/_classification.py

```diff
@@ -1586,52 +1657,37 @@ def precision_recall_fscore_support(
     )
 
     # warn for f-score only if zero_division is warn, it is in warn_for
-    # and BOTH prec and rec are ill-defined
+    # and BOTH precision and recall are ill-defined
```

Comment:
This can be reverted so a reviewer can focus on the changes that have to zero division = nan